modules/cluster-logging-deploy-eo-cli.adoc

```diff
@@ -184,14 +184,14 @@ The Elasticsearch Operator is installed to the `openshift-operators-redhat` Name
 oc get csv --all-namespaces
 
 NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
-default                                                 elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-node-lease                                         elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-public                                             elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-system                                             elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-apiserver-operator                            elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-apiserver                                     elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-authentication-operator                       elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-authentication                                elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
+default                                                 elasticsearch-operator.4.4.1-202002032140    Elasticsearch Operator   4.4.1-202002032140               Succeeded
```

Comment:
In this case, `4.4.1` isn't out yet. Also, the date is going to be different for `4.4.0`.

(I feel like what to do about version and date sensitive output is yet another discussion we can have...)